code/style.css

```diff
@@ -1,140 +1,148 @@
+    font-weight: 600;
+    line-height: 26px;
+    flex: 1;
 }
 
 main {
-  margin: 0 auto;
-  width: 100%;
-  max-width: 700px;
-  height: 600px;
-  border-radius: 30px;
-  background: #fff;
-  padding: 20px 24px;
-  padding-top: 0;
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
+    margin: 0 auto;
+    width: 100%;
+    max-width: 700px;
+    height: 600px;
+    border-radius: 30px;
+    background: #fff;
+    padding: 20px 24px;
+    padding-top: 0;
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    border: 1px solid #6155a6;
 }
 
 .chat {
-  flex: 1;
-  display: flex;
-  justify-content: flex-start;
-  overflow: scroll;
-  flex-direction: column;
-  padding-bottom: 16px;
+    flex: 1;
+    display: flex;
+    justify-content: flex-start;
+    overflow: scroll;
+    flex-direction: column;
+    padding-bottom: 16px;
 }
 
 .bot-msg {
-  display: flex;
-  margin: 16px 8px 0 0;
-  flex-shrink: 0;
+    display: flex;
+    margin: 16px 8px 0 0;
+    flex-shrink: 0;
 }
 
 .user-msg {
-  display: flex;
-  justify-content: flex-end;
-  margin: 16px 0 0 8px;
-  flex-shrink: 0;
+    display: flex;
+    justify-content: flex-end;
+    margin: 16px 0 0 8px;
+    flex-shrink: 0;
 }
 
 .bot-msg img,
 .user-msg img {
-  width: 60px;
-  height: 60px;
+    width: 60px;
+    height: 60px;
 }
 
 .bubble {
-  background: #e5e9ff;
-  font-weight: 600;
-  font-size: 16px;
-  line-height: 26px;
-  padding: 16px 24px;
-  color: #0026ff;
-  max-width: 40%;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 26px;
+    padding: 16px 24px;
+    max-width: 40%;
 }
 
 .bot-bubble {
-  border-radius: 0px 26px 26px 26px;
-  margin-left: 8px;
+    border-radius: 0px 26px 26px 26px;
+    margin-left: 8px;
+    background: #ffabe1
 }
 
 .user-bubble {
-  border-radius: 26px 0 26px 26px;
-  margin-right: 8px;
+    border-radius: 26px 0 26px 26px;
+    margin-right: 8px;
+    background: #a685e2
 }
 
 .input-wrapper {
-  display: flex;
-  justify-content: center;
+    display: flex;
+    justify-content: center;
 }
 
 .input-wrapper form {
-  width: 100%;
-  display: flex;
+    width: 100%;
+    display: flex;
+    border: 1px solid #a685e2;
+}
+
+input {
+    background-color: #fff;
+    color: #6155a6;
 }
 
 button {
-  background-color: #0026ff;
-  color: white;
-  border: none;
-  border-radius: 4px;
-  padding: 16px 40px;
-  margin-right: 4px;
-  font-size: 16px;
-  line-height: 26px;
-  font-family: 'Montserrat';
-  font-weight: 500;
-  cursor: pointer;
-  transition: all 0.3s ease;
+    background-color: #6155a6;
+    color: white;
+    border: none;
+    border-radius: 4px;
+    padding: 16px 40px;
+    margin-right: 4px;
+    font-size: 16px;
+    line-height: 26px;
+    font-family: 'Montserrat';
+    font-weight: 500;
+    cursor: pointer;
+    transition: all 0.3s ease;
 }
 
 button:hover {
-  opacity: 0.9;
-  transition: all 0.2s ease;
-}
+    opacity: 0.9;
+    transition: all 0.2s ease;
+}
```

Comment:
Please check your prettier settings, because you have a very big indentation in your CSS file. It should be set to `2 spaces` 🤓

Reply:
Thank you for your comment. The indentation of CSS looks fine on my VS Code.